curiouslearning/FeedTheMonsterJS
# fix - Cr -155 Development build and verify "CV Creole" FTM

# Changes
- lang/caboverdecreole
- and updated build folder

# How to test
- Check caboverdecreole in dropdown

Ref: [CR-155](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-155)


[CR-155]: https://curiouslearning.atlassian.net/browse/CR-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
